torch/nn/quantized/modules/wrapper_module.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import torch
+from torch._ops import ops
+from torch.nn import Module
+
+_FLOAT_MODULES = {
+    torch.add: ops.quantized.add,
+    torch.cat: ops.quantized.cat
+}
+
+
+"""Wrappes unary operators."""
+class UnaryWrapper(Module):
+    def __init__(self, operation):
+        super(UnaryWrapper, self).__init__()
+        self.operation = operation
+
+    def forward(self, x):
+        return self.operation(x)
```

Comment:
Need to have type annotation for this to work
```
class UnaryWrapper(nn.Module):
    def __init__(self, operation):
        super(UnaryWrapper, self).__init__()
        self.operation = operation

    def forward(self, x):
        # type: (List[Tensor]) -> Tensor
        return self.operation(x)
```

Reply:
This annotation is only valid for Cat (which takes the list). Other unary ops, shall we choose to quantize them, should have `Tensor` annotation.

I really think it's should be an implementation details. We can expose a single function `make_wrapper(torch.something)` and have internal dictionary dispatch to pick the right wrapper (out of a few - like binary, unary, list). All these wrappers aren't that generic anyway since _FLOAT_MODULES explicitly lists supported functions.